semgrep-core/src/runner/Run_semgrep.ml

```diff
@@ -546,14 +603,54 @@ let semgrep_with_raw_results_and_exn_handler config =
    * semgrep-core -rules <(curl https://semgrep.dev/c/p/ocaml) ...
    *)
   let rules_file = replace_named_pipe_by_regular_file rules_file in
+  let extractors_file =
+    if config.extractors_file <> "" then
+      Some (replace_named_pipe_by_regular_file config.extractors_file)
+    else None
+  in
   try
+    logger#linfo
+      (lazy (spf "Parsing %s:\n%s" rules_file (read_file rules_file)));
+    let timed_extractors =
```

Comment:
again move in helper function in the 'Extract helper' section to not clutter the original function.

Reply:
Will do.